src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.h

```diff
@@ -23,14 +23,17 @@
 #include <string>
 #include <vector>
 
+using ICEServerDecodableStruct = chip::app::Clusters::Globals::Structs::ICEServerStruct::DecodableType;
+using WebRTCSessionStruct      = chip::app::Clusters::Globals::Structs::WebRTCSessionStruct::Type;
+using ICECandidateStruct       = chip::app::Clusters::Globals::Structs::ICECandidateStruct::Type;
+using StreamUsageEnum          = chip::app::Clusters::Globals::StreamUsageEnum;
+using WebRTCEndReasonEnum      = chip::app::Clusters::Globals::WebRTCEndReasonEnum;
```

Comment:
Please, no global-scope using in headers.

Reply:
Moved it inside the WebRTCTransportProvider namespace.